examples/opendsme/Makefile

```diff
@@ -0,0 +1,47 @@
+# name of your application
+APPLICATION = opendsme_example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= nrf52840dk
```

Comment:
I haven't actively used ZEP, but seen it used for easier testing even in high-level parts. Should that just work, or is DSME too dependent on timings that would not be upheld by simulated 802.15.4?

Reply:
DSME is dependent on timings, but probably ZEP is fast enough to run DSME. I can give it a try.

Reply:
well, it compiles... I have no idea what to expect though. I will report back asap.

Reply:
hmmm it seems to be an easy to fix bug:
```
0x080713ca in ieee802154_radio_request_on (dev=0x809fc00 <_socket_zep_netdev>) at /home/jialamos/Development/RIOT/sys/include/net/ieee802154/radio.h:1037
1037        return dev->driver->request_on(dev);
```

Probably there's a null `dev` somewhere. I will investigate

Reply:
There are 2 problems with `native`:
1. One of the scheduling algorithms of openDSME uses `float`, which automatically makes it a victim of #495 
2. The RX timestamps are calculated for "real" IEEE 802.15.4 frames using RX Start and some symbol time calculations. To adapt it to `native` it will definitely require some steps there

Therefore, would you all be fine to postpone `native` support to the near future?

Reply:
For me, it would be.